### PR TITLE
Dedicated buttons for local/express mode

### PR DESCRIPTION
**Pull request by AaronVanGeffen**
Something that's been irking me for a while is that the local/express mode was part of the route/orders table, rather than a separate button. I think this makes the mode switching less discoverable and intuitive. In this PR, I propose changing it to use dedicated buttons instead.

From a code perspective, this also removes the need for various hacks to accommodate the first item in the list having a different function. This was not the primary motivation for this change, but definitely a nice bonus.

Old UI:
Screenshot (1)

New UI:
Screenshot (3)

### CODE REVIEW

**Comment by marijnvdwerf** [comment]
Could use some more spacing between the button and the list. Now it looks like those tableview headers.

**Comment by AaronVanGeffen** [comment]
Hmm, you're right. It didn't bother me much, but I've pushed it down a few pixels:

Old:
Screenshot (3)

New:
Screenshot (3)

Better?

**Review by marijnvdwerf — APPROVED** [review]

File: src/OpenLoco/Windows/Vehicle.cpp [hunk 1]
```diff
@@ -2596,36 +2622,24 @@ namespace OpenLoco::Ui::Vehicle
         static void getScrollSize(Ui::window* const self, const uint32_t scrollIndex, uint16_t* const width, uint16_t* const height)
         {
             auto head = Common::getVehicle(self);
-
-            // Space for the end of orders and express/local item
-            *height = 10 * 2;
-
             auto table = getOrderTable(head);
-            *height += 10 * std::distance(table.begin(), table.end());
+            *height = 10 * std::distance(table.begin(), table.end());
```
Comment (duncanspumpkin) [inline src/OpenLoco/Windows/Vehicle.cpp:2626]: You've removed the end of orders as well

Comment (AaronVanGeffen) [reply src/OpenLoco/Windows/Vehicle.cpp:2626]: An ironic regression. :) Thanks!

**Pull request merged**